### DISCUSSION TITLE
NAS Redirection Question

### DISCUSSION THREAD

**Discussion by DoubleTheAndreno** [Monika-After-Story/MonikaModDev#8137] [QA - tech support + general] [answered]
Lets say I was redirecting the game's save files to a NAS.  Right now I'm redirecting the %appdata% persistent folder and the characters folder.  

Is there anything else I should redirect before expecting the game to be able to be closed on one computer, opened on another, and have it remember everything between them?

**Comment by multimokia** [comment, marked as answer]
if anything I'd mirror the `game` folder too. The last thing you want is version mismatches as that can cause persistent corruption and/or event errors due to missing files/functions/etc.
